formats/xml/src/main/java/io/cloudevents/xml/XMLUtils.java

```diff
@@ -36,9 +36,15 @@
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 class XMLUtils {
 
+    private static final Pattern XML_PATTERN = Pattern.compile("^(application|text)\\/([a-zA-Z]+\\+)?xml(;.*)*$");
+    private static final Pattern TEXT_PATTERN = Pattern.compile("^application\\/([a-zA-Z]+\\+)?(xml|json)(;.*)*$");
```

Comment:
We're adding `json` too because it's a common / well known "text-based format", right?

Reply:
Yes ... any `text/*` content-types are caught in the utility function without using the regex (to keep it simpler!).